atom.go

```diff
@@ -0,0 +1 @@
+
```

Comment:
Based on your changes the following files may need to be updated:
- [ ] [Editor plugin FAQ]()
- [ ] [How to install Kite for Atom]()
- [ ] [Editor plugin technical spec]()
- [ ] [ProductSpec: Atom editor]()